doc/examples/reconst_qtiplus.py

```diff
@@ -0,0 +1,263 @@
+"""
+=====================================================================
+Applying positivity constraints to Q-space Trajectory Imaging (QTI+)
+=====================================================================
+
+Q-space trajectory imaging (QTI) [1]_ with applied positivity constraints
+(QTI+) is an estimation framework proposed by Herberthson et al. [2]_ which
+enforces necessary constraints during the estimation of the QTI model
+parameters.
+
+This tutorial briefly summarizes the theory and provides a comparison between
+performing the constrained and unconstrained QTI reconstruction in DIPY.
+
+Theory
+======
+
+In QTI, the tissue microstructure is represented by a diffusion tensor
+distribution (DTD). Here, DTD is denoted by $\mathbf{D}$ and the voxel-level
+diffusion tensor from DTI by $\langle\mathbf{D}\rangle$, where
+$\langle \ \rangle$ denotes averaging over the DTD. The covariance of
```

Comment:
This line does not render correctly in the documentation:

<img width="818" alt="image" src="https://user-images.githubusercontent.com/23106443/166960894-e2bdc16a-8fa3-4f3d-8d45-f398aa47172c.png">

Reply:
What is the problem here?

Reply:
Not sure what is going wrong but it does not recognize `$\langle \ \rangle$`.   

You can see above that the text is empty  around this sentence: `where
$\langle \ \rangle$ denotes averaging over the DTD.`

it might be the slash alone... not sure...

Reply:
If the snapshot you posted is how the doc is currently rendered, it should be correct, since we just want to use the empty <> parenthesis to say that those are used to denote averaging.

Reply:
ok, thank you for the feedback @DenebBoito